### ISSUE TITLE
Mute User(s)

### ISSUE THREAD

**Issue by ArchBlood** [humhub/humhub#4188]
### Is your feature request related to a problem? Please describe.
In the growing issues with bullying and abuse of social platforms, I believe that this is truly needed to stamp out a lot of hate that goes on in the world.

### Describe the solution you'd like
With the ability to mute a user they wouldn't be able to post/comment/like on any profile/space posts of the user that has muted them. Of course the permissions would be implemented into the user's profile and into the admin permissions for site & space owner and such, as it would be their posts that these kinds of things that would be happening on.

### Describe alternatives you've considered
This is a alternative to #564

**Comment by luke-** [comment]
I think when blocking the user they should also be muted.

Does it not make sense to combine the two issues?

**Comment by ArchBlood** [comment]
Well when you get down to the core functions of both they are meant to help stop a user from doing harm, but ideally both functions would be doing it in a different way, this is why I opened a separate issue. :+1:

**Comment by ArchBlood** [comment]
When I hear someone has gotten muted, this normally means they have no way of contacting anyone on the platform, whereas blocking is for individual people.

Muting can be extended far more, can be implemented into spaces where admins and mods have the ability to mute a rouge user causing issues as well as globally by site admin or those that have the permissions to use a site-wide mute.

Blocking should be thought of as between two users and one is harassing the other, the person who was blocked shouldn't be able to have any contact with that person that blocked them unless they are unblocked. I believe this would be an interesting idea, especially when dealing with space posts, but with how the profile stream now shows posts from spaces you've posted in this should be somewhat easier to work around. :thinking:

**Comment by luke-** [comment]
@ArchBlood Can we close this?
